### ISSUE TITLE
improvement suggestion

### ISSUE THREAD

**Issue by edo-jan** [itcon-pty-au/typingmind-cloud-backup#10]
First of all, thanks a lot for your app. It works great and sync everything to my S3 bucket. However, I am not too clear about the recovery process given that the backup is taken every 1 min.  
Let's take a scenario whereas I have accidently deleted / corrupted certain plugins, etc.  This corruption would immediately (within a minute) be propagated to S3 - correct?  I am wondering how do I restore the backup state to e.g. as of 10 min ago?  I guess, something like a time machine would be helpful to "Import from S3" to be used in conjunction with a time and date option to select.  It appears that currently, the moment .json is overwritten the only one to use would be the one from the previous date. Am I wrong?  

I wonder if there is a simpler way of achieving the same?  Also I would much appreciate if you could shed some light on how this works with the built-in cloud backup feature (10mg of space which comes with the premium version). Currently, my cloud storage is almost full and I am not sure what would be a recommended way to use it in conjunction with your plugin so that both are used without a conflict.

**Comment by itcon-pty-au** [comment]
You are correct, the only option you have is to restore using the previous day's backup. This is mainly for restoring data in case of a catastrophic mistake. The time machine is a good idea. I may not be able to implement it the way you envisioned it. But let me think about what can be done.

This plugin does work along with the built in cloud backup feature. Did you face any conflicts when using both together? However, an export from the inbuilt cloud backup cannot be restored using this extension as the backup file formats are different currently.
